src/models/patient.ts

```diff
@@ -0,0 +1,44 @@
+import mongoose,{Schema,Model,Document} from "mongoose";
+
+export enum PetType {
+    cat = "cat",

```

Comment:
here the value is in a smaller case but in the case of currency it was upper so make it consistent. the key should be Upper case for enums